lib/bambora/v1/payment_resource.rb

```diff
@@ -43,7 +40,7 @@ def initialize(client:)
       #
       # @return [Hash] Indicating success or failure of the operation.
       def create(payment_data)
-        @client.request(method: :post, path: @sub_path, body: payment_data)
+        client.post(path: sub_path, body: payment_data, api_key: api_key)
```

Comment:
Look at the difference here:

```ruby
client.request(method: :post, ...)
```

versus

```ruby
client.post(...)
```

Much more terse and clear, I love it.